products/cakephp.md

```diff
@@ -6,9 +6,8 @@ permalink: /cakephp
 alternate_urls:
 -   /cake-php
 -   /cake
-releasePolicyLink: https://book.cakephp.org/4/en/release-policy.html
+releasePolicyLink: https://github.com/cakephp/cakephp/security/policy#supported-versions
```

Comment:
This field name is confusion on endoflife.date. IMHO it should be named `supportPolicyLink` because that's the "more information" link displayed at the end of the article that is supposed to give more details on how the product is managing its support, EOL dates, and versions.

Linking to the security policy is not the best choice either. I think I should have linked to https://github.com/cakephp/cakephp/wiki because it gives at least informations on 2.x and 3.x EOL date. Do you think it is possible to also document [what's planned for 4.x and 5.x](https://bakery.cakephp.org/2023/01/01/cakephp_3_end_of_life_3105_released.html) as well.

> Going forward the CakePHP team will continue to support 4.x and develop 5.x. Once 5.0.0 is released, 4.x will receive bugfixes for 24 months, and security fixes for 36 months.

Reply:
Closing this conversation, discussion moved to https://github.com/endoflife-date/endoflife.date/pull/2289.